controllers/admin/AdminStoresController.php

```diff
@@ -333,7 +333,7 @@ public function postProcess()
             $langs = Language::getLanguages(false);
             /* Cleaning fields */
             foreach ($_POST as $kp => $vp) {
-                if (!in_array($kp, ['checkBoxShopGroupAsso_store', 'checkBoxShopAsso_store', 'hours'])) {
```

Comment:
@jf-viguier If you do this, then you also disable trim capabilities on not-string fields such as numeric inputs.

I'm not sure whether that's an issue 🤔  do the numeric inputs in the Stores BO page (ZIP code, longitude, latitude) behave OK ? I guess yes because they are passed as strings from $_POST.

What else can we get from $_POST ? Only strings and arrays ?

Reply:
@matks $_POST values are always strings or array of strings I guess. Integer casting is done after the trim.
By the way there are number_format for lat/long, (int) casting for ids. Postcode is a string (think about Corsica)

Reply:
This change seems OK then.

But I have another one 😁 : previous implementation was NOT trimming if `$kp` was one of 'checkBoxShopGroupAsso_store', 'checkBoxShopAsso_store' or 'hours'.

If I am not wrong, with your new implementation you DO trim these 3 values.

So in order to keep the "dont trim these 3" we need to do keep the "exclusion list" but _add_ the string type check:
```
if (!in_array($kp, ['checkBoxShopGroupAsso_store', 'checkBoxShopAsso_store', 'hours']) && is_string($kp)) {
```

Reply:
No, these 3 ones are arrays

Reply:
Haha 😄  so it works

Thanks for answering, I approve the PR